wordlister.py

```diff
@@ -66,65 +62,36 @@ def get_input_words(self, file_path: str) -> set:
+        The words are then added to the set which will contain all the generated words.
 
         :param perms: tuple containing a permutation of the word list that must be elaborated
         :type perms: tuple
-        :return data: set
         """
         
-        result = set()
-        output_file = 'tmp.txt'
-        
         if len(set(map(str.lower, permutation))) == len(permutation):
             line_printer = ''.join(permutation)
             if self.args.min <= len(line_printer) <= self.args.max:
-                result.add(line_printer + '\n')
+                self.wordlist.add(line_printer + '\n')
                 if self.args.append is not None and len(line_printer) + len(self.args.append) <= self.args.max:
-                    result.add(f'{line_printer}{self.args.append}\n')
+                    self.wordlist.add(f'{line_printer}{self.args.append}\n')
                 if self.args.prepend is not None and len(line_printer) + len(self.args.prepend) <= self.args.max:
-                    result.add(f'{self.args.prepend}{line_printer}\n')
+                    self.wordlist.add(f'{self.args.prepend}{line_printer}\n')
                 if self.args.leet is True:
-                    result.update(self.leet_and_append_and_prepend(line_printer))
-    
-        with open(output_file, 'a') as f_out:
-            f_out.writelines(result)
-                   
-        return output_file
+                    self.wordlist.update(self.leet_and_append_and_prepend(line_printer))
 
 
     def run(self):
         input_words = self.get_input_words(self.args.input)
         for x in range(self.args.perm):
-            output_iterable = map(self.printer, permutations(input_words, x + 1))
+            for perm in permutations(input_words, x + 1):
+                self.printer(perm)
         
-        # Remove duplicates leveraging sets' properties
-        output_files = set(output_iterable)
-        
-        # If we are on linux we leverage GNU tools to clean the output efficiently
-        if 'linux' in platform:
-            # read all workers' output files
-            p1 = Popen(['cat', *output_files], stdout=PIPE)
-            # sort the output of p1 and remove duplicated lines
-            p2 = Popen(['sort', '-u', '-o', f'{self.args.output}'], stdin=p1.stdout, stdout=PIPE)
-            p2.communicate()
-        # else we do it with Python's sets
-        else:
-            uniq_output = set()
-            with open(self.args.output, 'w') as f_out:
-                for file in output_files:
-                    with open(file, 'r') as f_in:
-                        uniq_output.update(f_in.read().splitlines())
-                
-                for line in uniq_output:
-                    f_out.write(line + '\n')
-    
-        for file in output_files:
-            # delete file after copying it to final output file
-            remove(file)
+        self.wordlist = sorted(self.wordlist)    
```

Comment:
Would it be possible to please make this sorting optional by defining a dedicated command line argument?